netutils/webclient/webclient.c

```diff
@@ -132,6 +129,37 @@
  * Private Types
  ****************************************************************************/
 
+enum webclient_state
+  {
+    WEBCLIENT_STATE_SOCKET,
+    WEBCLIENT_STATE_CONNECT,
+    WEBCLIENT_STATE_PREPARE_REQUEST,
+    WEBCLIENT_STATE_SEND_REQUEST,
+    WEBCLIENT_STATE_SEND_REQUEST_BODY,
+    WEBCLIENT_STATE_STATUSLINE,
+    WEBCLIENT_STATE_HEADERS,
+    WEBCLIENT_STATE_DATA,
+    WEBCLIENT_STATE_CLOSE,
+    WEBCLIENT_STATE_DONE,
+  };
+
+struct conn
+{
+  bool tls;
+
+  /* for !tls */
+
+  int sockfd;
+  unsigned int flags;
+
+#define CONN_WANT_READ  WEBCLIENT_POLL_INFO_WANT_READ
+#define CONN_WANT_WRITE WEBCLIENT_POLL_INFO_WANT_WRITE
```

Comment:
Placement of these preprocessor definitions is not compliant to the coding standard.

Reply:
fixed